aemedge/blocks/link-list/link-list.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Link List Block
+ */
+
+/**
+ * Wrap the text node of a link with a `span` element.
+ * (!) Text node can also be positioned before the icon.
+ * @param link {Element} The link element
+ */
+function moveLinkTextNodeToSpan(link) {
+  Array.from(link.childNodes)
```

Comment:
Wait, wrapping in span can be done via decroateIcons, we would not need this function?

Reply:
@mhaack `decorateIcons` is used automatically because of the `:icon:` inside the Word content. I need this function, because the icon can be placed on the left and the right side of the link text. Default behavior is plain text + icon (in span). You have no possibility to place margin to the left with text nodes. So I want to span the text to be able to margin to the right.